enterprise/models.py

```diff
@@ -2733,13 +2733,38 @@ def __repr__(self):
         """
         return self.__str__()
 
+    @classmethod
+    def validate_compression(cls, enable_compression, data_type, delivery_method):
+        """Check enable_compression flag is set as expected
```

Comment:
Please update doc string format according to https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings